### PR TITLE
[instant] Header tweaks

### PR DESCRIPTION
**Pull request by steveklebanoff**
## Description

- Right align amounts
- Autofocus input

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Comment by BMillman19** [comment]
Looks like it stops autofocusing if you switch to a different asset.

Repro steps:

* select a token
* observe the input is focused
* open token picker
* select a different token
* observe the input is no longer focused

**Comment by steveklebanoff** [comment]
Ah snap, thought this was working -- will fix.  Thanks for testing it out!

**Comment by steveklebanoff** [comment]
@BMillman19  This should be fixed now, see https://github.com/0xProject/0x-monorepo/pull/1294/commits/10c9d0b723ba190a7f1c4df112ae0c5b37887bb9 . Approach is a little funky, but discussed with @fragosti and he was on board with it

File: packages/instant/src/components/scaling_input.tsx [hunk 1]
```diff
@@ -136,6 +138,7 @@ export class ScalingInput extends React.Component<ScalingInputProps, ScalingInpu
                 width={this._calculateWidth(phase)}
                 maxLength={maxLength}
                 disabled={isDisabled}
+                autoFocus={hasAutofocus}
```
Comment (fragosti) [inline packages/instant/src/components/scaling_input.tsx]: 👍

**Pull request merged**